pynopegl-utils/pynopegl_utils/viewer/config.py

```diff
@@ -73,7 +74,11 @@ class EncodeProfile:
 
 
 class Config(QtCore.QObject):
-    FILEPATH = op.join(os.environ.get("XDG_DATA_HOME", op.expanduser("~/.local/share")), "nope.gl", "viewer.json")
+    FILEPATH = (
+        Path(os.environ.get("XDG_DATA_HOME", Path.home().joinpath(".local", "share")))
+        .joinpath("nope.gl", "viewer.json")
```

Comment:
You can do `Path.home() / ".local" / "share"` (same with the others `.joinpath`)

Reply:
Done.